### PR TITLE
added handling when search contains faulty inputs [#158718086]

### PR DESCRIPTION
**Pull request by alarku07**

### CODE REVIEW

**Review by timo-piirmann — APPROVED** [review]

File: grails-app/i18n/messages.properties [hunk 1]
```diff
@@ -34,6 +34,7 @@ search.header.kingdom.Fungi = Fungi
 search.header.kingdom.Plantae = Plants
 search.header.kingdom.Protista = Protists
 search.download.link.title = Download a list of taxa for your search
+search.errorMsg = Search failed. Please check your inputs and try again
```
Comment (raivop) [inline grails-app/i18n/messages.properties]: Period's missing.

File: grails-app/i18n/messages_et.properties [hunk 2]
```diff
@@ -34,6 +34,7 @@ search.header.kingdom.Fungi = Seened
 search.header.kingdom.Plantae = Taimed
 search.header.kingdom.Protista = Protistid
 search.download.link.title = Laadi leitud taksonite nimekiri alla
+search.errorMsg = Otsing ebaõnnestus. Kontrollige filtrite väärtuseid ja proovige uuesti
```
Comment (raivop) [inline grails-app/i18n/messages_et.properties]: Missing a period.

File: grails-app/controllers/au/org/ala/bie/SpeciesController.groovy [hunk 3]
```diff
@@ -70,26 +70,34 @@ class SpeciesController {
 
         if (filterQuery.size() > 1 && filterQuery.findAll { it.size() == 0 }) {
             // remove empty fq= params IF more than 1 fq param present
-            def fq2 = filterQuery.findAll { it } // excludes empty or null elements
+            def fq2 = filterQuery.findAll { it }  // excludes empty or null elements
```
Comment (raivop) [inline grails-app/controllers/au/org/ala/bie/SpeciesController.groovy]: y tho?

Comment (alarku07) [reply grails-app/controllers/au/org/ala/bie/SpeciesController.groovy]: habit from plutof backend style

File: grails-app/i18n/messages.properties (hunk 1 above)
Comment (alarku07) [reply grails-app/i18n/messages.properties]: look around. do you see any ending periods anywhere else?

File: grails-app/i18n/messages_et.properties (hunk 2 above)
Comment (alarku07) [reply grails-app/i18n/messages_et.properties]: nah

File: grails-app/i18n/messages.properties (hunk 1 above)
Comment (raivop) [reply grails-app/i18n/messages.properties]: This is different becasue you have two sentences here. If one's ending with a period, so should the other.

You don't have to use periods if the element consists of one label or statement.

Comment (alarku07) [reply grails-app/i18n/messages.properties]: the last sentence doesn't need to end with a period

**Pull request merged**